lib/validation.js

```diff
@@ -22,5 +24,9 @@ module.exports = function rangeValidation(input, settings) {
     return `${input.target.name} cannot be left blank!`;
   }
 
+  if (input.target.value < settings.min || input.target.value > settings.max) {
+    return `${input.target.name} must be a value within given range!`;
```

Comment:
@rachelnicole same with this one. Please write a test which checks for a variable outside the current range.
Code coverage test:
https://travis-ci.org/punchcard-cms/input-plugin-range/builds/132905940
you can see where it shows line 28 is not covered:
```
-------------------------|----------|----------|----------|----------|----------------|
File                     |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
-------------------------|----------|----------|----------|----------|----------------|
 input-plugin-range/     |      100 |      100 |      100 |      100 |                |
  index.js               |      100 |      100 |      100 |      100 |                |
 input-plugin-range/lib/ |    83.33 |     87.5 |      100 |    83.33 |                |
  validation.js          |    83.33 |     87.5 |      100 |    83.33 |             28 |
-------------------------|----------|----------|----------|----------|----------------|
All files                |     87.5 |     87.5 |      100 |     87.5 |                |
-------------------------|----------|----------|----------|----------|----------------|
```